.github/workflows/build.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: ghcr.io/${{ github.repository }}
```

Comment:
The [pipeline that this is replacing](https://github.com/lsst-dm/jenkins-dm-jobs/blob/main/pipelines/sqre/infra/build_docker_gc.groovy#L45-L70) adds a `latest` tag and defeats the build cache.  Not sure if we need those here.